docs/revsets.md

```diff
@@ -109,6 +109,9 @@ revsets (expressions) as arguments.
   `present(HEAD@git)`.
 * `visible_heads()`: All visible heads (same as `heads(all())`).
 * `root()`: The virtual commit that is the oldest ancestor of all other commits.
+* `immutable()`: Commits that shouldn't be rewritten. Defined as the ancestors
+  of the revset configured in `revsets.immutable-heads`, as well as the `root()`
```

Comment:
Optional: I feel like if you swap the order and write "the `root()` commit and the ancestors of the revset ...", you can get rid of the parenthetical clause. Or you could replace it with "(if non-empty)".

Reply:
Done.